docs/topic/features.md

```diff
@@ -34,9 +34,26 @@ the data. This is very commonly used by organizations that provide big datasets
 on Google Cloud storage, to sustainably share costs of maintaining the data.
 
 When this feature is enabled, users on a hub accessing cloud buckets from
-other organizations marked as 'requestor pays' will increase our cloud bill.
+other organizations marked as 'Requester Pays' will increase our cloud bill.
 Hence, this is an opt-in feature.
 
+```{important}
+This feature enables the hub users to access `Requester Pays` buckets,
+**outside** of their project.
```

Comment:
Here, we talk about using requester pays so hub users can access buckets **outside** their hub... (thought continued in another comment)